apps/tng-dictionary/src/components/VocabularyListIndex/VocabularyListIndex.tsx

```diff
@@ -6,12 +6,25 @@ import Loading from '../Loading/Loading';
 import './VocabularyListIndex.module.css';
 import { ThemeProvider, createTheme } from '@mui/material/styles';
 import { motion } from 'framer-motion';
-import { TextField } from '@mui/material';
+import { TextField, CircularProgress } from '@mui/material';
 import SearchIcon from '@mui/icons-material/Search';
 import MenuBookTwoToneIcon from '@mui/icons-material/MenuBookTwoTone';
+import doValuesMatchFilters from '../../utilities/doValuesMatchFilters';
+import Stack from '@mui/material/Stack';
 
 
 
+type appState = {
+  vocabularyLists: HasIdAndName[];
+  searchContext: 'name'
+}
+
+const stringIncludes = (input: string, textToMatch: string) => input.includes(textToMatch)
```

Comment:
I wouldn't copy and paste here. Let's break out the `stringIncludes` function to a `utilities` directory instead and import it.